package.json

```diff
@@ -17,7 +17,7 @@
     "body-parser": "1.20.3",
     "express": "4.21.1",
     "express-winston": "2.5.1",
-    "integrator-extension": "0.3.11",
+    "integrator-extension": "0.3.12",
```

Comment:
I hope there are no other changes in this version that we might have to check.

Reply:
There are no code changes made apart from this change.